### ISSUE TITLE
Can not select Theme dracula

### ISSUE THREAD

**Issue by satorope** [JanDeDobbeleer/oh-my-posh#2154]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

A bug happened!

I installed Oh My Posh from Winget.
I cannot select the Dracula theme.

### Theme

oh-my-posh init pwsh --config "C:\Program Files (x86)\oh-my-posh\themes\dracula.omp.json" | Invoke-Expression

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
Oh My Posh Error:

open C:\Program Files (x86)\oh-my-posh\themes\dracula.omp.json.bak: Access is denied.
```

**Comment by lewis-yeung** [comment]
The `C:\Program Files (x86)\oh-my-posh\themes\dracula.omp.json.bak` seems to be a backup saved by auto-migration. What happens if you run `oh-my-posh init pwsh --config "C:\Program Files (x86)\oh-my-posh\themes\dracula.omp.json" | Invoke-Expression` as admin, which means in an elevated PowerShell session?

**Comment by JanDeDobbeleer** [comment]
@satorope oh-my-posh needs to migrate your theme but can't as it's not accessible from a rights perspective. You need to move your custom theme out of that location so oh-my-posh can migrate it (or use the shipped version, but somehow the executable `oh-my-posh.exe` and this theme are not in sync).

**Comment by satorope** [comment]
open C:\Program Files (x86)\oh-my-posh\themes\dracula.omp.json.bak: Access is denied.

I don't remember if I first started PowerShell with an administrator view.
"C:\Program Files (x86)\oh-my-posh\themes\dracula.omp.json.bak" does not exist.
I have also searched for dracula.omp.json.bak on my entire pc and it does not exist as well.
Nor have I ever created a custom theme.
But every time I config or run Get-Poshtheme I get this error.
Uninstalling oh-my-posh and then restarting my PC and reinstalling oh-my-posh does not change it.

I could not configure it from a legitimate installation.
I was finally able to set it up by copying the dracula.omp.json to my mydocument.
